plugins/pulp_rpm/plugins/importers/yum/depsolve.py

```diff
@@ -163,9 +155,6 @@ def fills_requirement(self, unit):
         :return:        True if the unit satisfies the Requirement, False otherwise
         :rtype:         bool
         """
-        if self.name != unit.name:
```

Comment:
I may have been too aggressive in trying to get non-Name (e.g. "npm(ansi) < 0.3") requires/provides to work but I'll have to check.  This code did manage to whitelist npm and its many nodejs deps from EPEL without having to list them all explicitly in Katello but now I'm wondering how I didn't end up with tons of extra RPMs.  I'll have to do some more testing.